opaleye-sqlite/src/Opaleye/SQLite/Internal/PackMap.hs

```diff
@@ -40,7 +40,7 @@ import qualified Data.Functor.Identity as I
 -- 'ProductProfunctor') in @s@ and @t@.  It is unclear at this point
 -- whether we want the same @Traversal@ laws to hold or not.  Our use
 -- cases may be much more general.
-data PackMap a b s t = PackMap (Applicative f =>
+data PackMap a b s t = PackMap (forall f. Applicative f =>
```

Comment:
Yeah, that's something that had to change in Opaleye a while ago but I didn't fix the sqlite version at the same time.